### ISSUE TITLE
Child profilePicture cannot be downloaded when using parent credentials

### ISSUE THREAD

**Issue by gruijter** [simplyGits/MagisterJS#74]
Below code works when the child credentials are used. But when using the parent login it doesn't work when trying to download the profilePicture from the same child.

```
		request.get({
			url: item.profileInfo().profilePicture(128, 128, false),
			encoding: null,
			headers: {
				cookie: item.http._cookie,
				'X-API-Client-ID': '12D8',
			},
		}).pipe(fs.createWriteStream(`./userdata/${item.profileInfo().id()}.jpg`));
```

**Comment by lesander** [comment]
Please define `doesn't work`. What does Magister's API return?

**Comment by lieuwex** [comment]
I guess you need to use the cookie of the Magister instance of the parent.

**Comment by lieuwex** [comment]
Worth noting that you don't have to do cookie magic anymore with v2.

**Comment by gruijter** [comment]
in v2 the issue resurfaces. see possible fix #87
